File: Ruby/k08_1sub.rb
def get_max_value(v1, v2, v3)
  temp = v1 >= v2 ? v1 : v2
  temp = temp >= v3 ? temp : v3
  temp
end

def get_min_value(v1, v2, v3)
  temp = v1 < v2 ? v1 : v2
  temp = temp < v3 ? temp : v3
  temp
end

File: Python/k08_1sub.py
def get_max_value(v1, v2, v3):
    temp = v1 if v1 >= v2 else v2
    temp = temp if temp >= v3 else v3
    return temp

def get_min_value(v1, v2, v3):
    temp = v1 if v1 < v2 else v2
    temp = temp if temp < v3 else v3
    return temp
